android/tangram/jni/platform_android.cpp

```diff
@@ -245,20 +256,25 @@ unsigned char* bytesFromFileSystem(const char* _path, unsigned int* _size) {
 
 }
 
-std::string stringFromFile(const char* _path, PathType _type) {
+std::string stringFromFile(const char* _path, PathType _type, const char* _resourceRoot) {
 
     unsigned int length = 0;
-    unsigned char* bytes = bytesFromFile(_path, _type, &length);
+    unsigned char* bytes = bytesFromFile(_path, _type, &length, _resourceRoot);
     std::string out(reinterpret_cast<char*>(bytes), length);
     free(bytes);
 
     return out;
 
 }
 
-unsigned char* bytesFromFile(const char* _path, PathType _type, unsigned int* _size) {
+unsigned char* bytesFromFile(const char* _path, PathType _type, unsigned int* _size, const char* _resourceRoot) {
```

Comment:
Would it be possible to load all scene resources *through* Scene and thereby keep resourceRoot internal to Scene?
Then we could remove all the PathType::resource special handling.